fedimint-bin-tests/src/main.rs

```diff
@@ -1109,6 +1124,15 @@ struct Args {
     command: Cmd,
 }
 
+async fn write_ready_file<T>(result: Result<T>) -> Result<T> {
+    let ready_file = env::var("FM_READY_FILE")?;
+    match result {
+        Ok(_) => fs::write(ready_file, "READY").await?,
```

Comment:
I just banned these via semgrep like yesterday or day before. :D

Reply:
does CI check precommit hooks? 
I don't run precommit hooks locally because my emacs just lags _a lot_.

Reply:
looks like you banned std::fs::write, not tokio::fs::write